src/test/java/pl/javastart/streamstask/StreamsTaskTest.java

```diff
@@ -0,0 +1,149 @@
+package pl.javastart.streamstask;
+
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class StreamsTaskTest {
+    StreamsTask streamsTask = new StreamsTask();
+
+    @Test
+    void shouldReturnEmptyList() {
```

Comment:
Fajnie gdyby nazwa metody oddawała to, co faktycznie testujemy. Tutaj lepszą nazwą byłoby "shouldNotFindWomen". EmptyList jest trochę zbyt ogólne